modules/services/window-managers/i3-sway/sway.nix

```
@@ -289,11 +289,11 @@ in {
     enable = mkEnableOption "sway wayland compositor";
 
     package = mkOption {
-      type = types.package;
+      type = with types; nullOr package;
       default = defaultSwayPackage;
       defaultText = literalExample "${pkgs.sway}";
       description = ''
-        Sway package to use. Will override the options 'wrapperFeatures', 'extraSessionCommands', and 'extraOptions'.
+        Sway package to use. Will override the options 'wrapperFeatures', 'extraSessionCommands', and 'extraOptions'. Set to <code>null</code> to not add any sway package to your path. This should be done if you want to use the nixos sway module to install sway.
```

I think this becomes a bit more readable if the line width is limited:
```suggestion
        Sway package to use. Will override the options
        'wrapperFeatures', 'extraSessionCommands', and 'extraOptions'.
        Set to <code>null</code> to not add any Sway package to your
        path. This should be done if you want to use the NixOS Sway
        module to install Sway.